apps/obs-web/src/views/DebrieferDcs.vue

```diff
@@ -0,0 +1,310 @@
+        <p style="font-weight: bold; font-size: 20px; letter-spacing: .5px;">DCS/AFI - {{ observerName }}</p>
+      </div>
+      <div class="col">
+        <q-select v-model="selectedYear" :options="yearOptions" outlined label="year" dense options-dense style="max-width: 300px"></q-select>
+      </div>
+      <div class="col">
+        <q-select v-model="selectedMonth" :options="['All', 'January', 'February', 'March', 'April', 'May', 'June', 'July', 'August', 'September', 'October', 'November', 'December']" outlined label="month" dense options-dense style="max-width: 300px"></q-select>
+      </div>
+      <div class="col" style="text-align: right">
+        <q-btn round icon="refresh" size="sm" color="primary" style="position: relative; top: 4px; margin-right: 10px" @click="refresh"></q-btn>
+        <q-btn
+          size="md"
+          color="primary"
+          dense
+          style="position: relative; top: 3px"
+          @click="addDcsRow"
+          >add dcs row</q-btn
+        >
+      </div>
+    </div>
+    <prime-table
+      :value="dcsRows"
+      :columns="dcsColumns"
+      type="DCS"
+      uniqueKey="_id"
+      :enableSelection="true"
+      :isFullSize="true"
+      :loading="loading"
+      @save="save"
+    >
+    </prime-table>
+  </div>
+</template>
+
+<script lang="ts">
+import {
+  createComponent,
+  ref,
+  onMounted,
+  watch,
+  computed,
+  onBeforeMount,
+} from '@vue/composition-api';
+import moment from 'moment';
+
+import { couchService } from '@boatnet/bn-couch';
+import { Client } from 'davenport';
+import { NoSubstitutionTemplateLiteral } from 'typescript';
+import { authService } from '@boatnet/bn-auth/lib';
+
+export default createComponent({
+  props: {
+    showData: Boolean,
+    isFullSize: Boolean,
+  },
+  setup(props, context) {
+    const store = context.root.$store;
+    const state = store.state;
+    const masterDB: Client<any> = couchService.masterDB;
+
+    const dcsColumns = [
+      {
+        field: 'tripNum',
+        header: 'Trip #',
+        key: 'tripNum',
+        width: '60',
+        isEditable: true,
+        type: 'text',
+      },
+      {
+        field: 'haulNum',
+        header: 'Haul #',
+        key: 'haulNum',
+        width: '60',
+        isEditable: true,
+        type: 'text',
+      },
+      {
+        field: 'collectionMethod',
+        header: 'Collection Method',
+        key: 'collectionMethod',
+        width: '80',
+        isEditable: true,
+        type: 'toggle',
+        listType: 'template',
+        list: ['OPTECS', 'Deck Form', 'hybrid'],
+      },
+      {
+        field: 'createdDate',
+        header: 'Date Added',
+        key: 'createdDate',
+        width: '80',
+        isEditable: true,
+        type: 'date'
+      },
+      {
+        field: 'level',
+        header: 'Level',
+        key: 'level',
+        width: '60',
+        isEditable: true,
+        type: 'toggle',
+        listType: 'template',
+        list: [
```

Comment:
You could use Object.values(TripLevel) here to get the values you defined in your enum. Same for the rest of the enums

Reply:
implemented